src/core/checkedint.d

```diff
@@ -682,10 +682,27 @@ unittest
 
 /// ditto
 pragma(inline, true)
+ulong mulu(uint x, ulong y, ref bool overflow)
+{
+    ulong r = x * y;
+    if (y >> 32 && x && (r / x) != y)
```

Comment:
@andralex this is D's coverage output which will insert `000001` as soon as this line is hit (no condition coverage).

Other languages do support this, e.g.
https://github.com/codecov/support/issues/227